tests/helpers/utils.py

```diff
@@ -0,0 +1,109 @@
+from pyscipopt import Model, quicksum, SCIP_PARAMSETTING, exp, log, sqrt, sin
+from typing import List
+
+def random_MIP_1():
```

Comment:
the `_1`s look strange when there's no `_2` 😄

Reply:
In my defense, it's still a draft :p

Reply:
Fair enough 😄

Reply:
Okay, my defense stopped working @mmghannam :D 

My new defense is that this can be something that we keep updating as we come across or come up with interesting models. What do you think? This PR is something that should only affect the developers.